iio/iio_types.h

```diff
@@ -45,6 +45,7 @@
 
 #include <stdbool.h>
 #include <stdint.h>
+#include <stdio.h>
```

Comment:
What is it needed for ? I don't see any other change in iio_types.h, so I suspect that it is needed only for something in a .c file, in which case it is more suitable to move it to that .c file

Reply:
Actually the file needed here is  "types.h".